common/reviews/api/rush-lib.api.md

```diff
@@ -1147,12 +1151,12 @@ export class RushConfiguration {
     readonly suppressNodeLtsWarning: boolean;
     // @beta
     readonly telemetryEnabled: boolean;
-    readonly tempShrinkwrapFilename: string;
-    readonly tempShrinkwrapPreinstallFilename: string;
```

Comment:
@william2958 Removing `tempShrinkwrapFilename` and `tempShrinkwrapPreinstallFilename` is a breaking API change.

How do you want to handle this? One idea would be to make the API as `@deprecated` and for the property getter to throw an exception when the subspaces feature is enabled.

Reply:
marked as @deprecated and throws exception when subspaces is enabled